lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/UpdateTaskHost.cs

```diff
@@ -54,8 +56,10 @@ internal sealed class UpdateTaskHost<TMapInput, TMapOutput, TResult> : TaskHostB
         /// <param name="taskCloseCoordinator">Task close Coordinator</param>
         /// <param name="invokeGc">Whether to call Garbage Collector after each iteration or not</param>
         /// <param name="taskId">task id</param>
+        /// <param name="contextRuntime">Reference of ContextRuntime id</param>
         [Inject]
         private UpdateTaskHost(
+            ContextRuntime contextRuntime,
```

Comment:
This looks like it breaks the REEF abstractions. Why would a Task get access to `ContextRuntime`, which is an internal implementation detail of the Evaluator?